api/src/legal-documents/application/api/legal-documents-api.js

```diff
@@ -1,16 +1,29 @@
 import { usecases } from '../../domain/usecases/index.js';
 
 /**
- * Accept legal document by user id.
+ * Accepts a legal document for a user by their ID.
  *
- * @param{string} params.service
- * @param{string} params.type
- * @param{string} params.userId
- *
- * @returns {Promise<void>}
+ * @param {Object} params - The parameters.
+ * @param {string} params.type - The type of the legal document. (e.g. 'TOS')
+ * @param {string} params.service - The service associated with the legal document. (e.g. 'pix-orga')
+ * @param {string} params.userId - The ID of the user.
+ * @returns {Promise<void>} - A promise that resolves when the legal document is accepted.
  */
 const acceptLegalDocumentByUserId = async ({ type, service, userId }) => {
   return usecases.acceptLegalDocumentByUserId({ type, service, userId });
 };
 
-export { acceptLegalDocumentByUserId };
+/**
+ * Gets the status of a legal document for a user by their ID.
+ *
+ * @param {Object} params - The parameters.
+ * @param {string} params.type - The type of the legal document. (e.g. 'TOS')
+ * @param {string} params.service - The service associated with the legal document. (e.g. 'pix-orga')
+ * @param {string} params.userId - The ID of the user.
+ * @returns {Promise<LegalDocumentStatus>} - A promise that resolves with the status of the legal document.
+ */
+const getLegalDocumentStatusByUserId = async ({ type, service, userId }) => {
+  return usecases.getLegalDocumentStatusByUserId({ type, service, userId });
+};
```

Comment:
Ça ne change rien à l'API, juste à la manière dont c'est écrit, cela fait une correspondance plus directe dans l'esprit quand on lit en mettant les conditions les plus importantes en premier et les moins discriminantes à la fin : 

```suggestion
const getLegalDocumentStatusByUserId = async ({ userId, service, type }) => {
  return usecases.getLegalDocumentStatusByUserId({ userId, service, type });
};
```

Reply:
Bien sûr il faudrait alors modifier les autres déclarations de fonctions comme `acceptLegalDocumentByUserId`.